.pipelines/OneBranch.Official.yml

```diff
@@ -86,4 +86,21 @@ extends:
             python -m pip install cibuildwheel
             python -m cibuildwheel --platform windows --output-dir $(REPOROOT)\out
           displayName: Build wheels
+        - script: |
+            7z x $(REPOROOT)\out\*.whl -o$(REPOROOT)\out\*
+          displayName: unzip the package
+        - task: onebranch.pipeline.signing@1
+          inputs:
+            command: 'sign'
+            signing_profile: 'external_distribution'
+            files_to_sign: '**/**/*.pyd;**/**/*.dll'
+            search_root: '$(REPOROOT)\out'
```

Comment:
'out' happens to be the default native build target folder, can we pick another name to avoid the potential conflict in the future?

Reply:
You can change the line 62:
```
          ob_outputDirectory: '$(REPOROOT)\out'
```

Reply:
I did not quite unterstand. Should I modify the output folder of wheel build task ? ```python -m cibuildwheel --platform windows --output-dir $(REPOROOT)\out```